src/csv.cr

```diff
@@ -111,16 +111,25 @@ class CSV
 
   # Builds a CSV. This yields a `CSV::Builder` to the given block.
   #
+  # Takes optional *quoting* argument to define quote behavior.
+  #
   # ```
   # result = CSV.build do |csv|
   #   csv.row "one", "two"
   #   csv.row "three"
   # end
   # result # => "one,two\nthree\n"
+  # result = CSV.build(quoting: CSV::Builder::Quoting::ALL) do |csv|
```

Comment:
`quoting: :all`

Reply:
Wow, I didn't know such a cool syntax! Thank you!